GitCommands/Settings/AppSettings.cs

```diff
@@ -1557,6 +1567,13 @@ public static bool RememberShowEntireFilePreference
             set => SetBool("RememberShowEntireFilePreference", value);
         }
 
+        public static BoolRuntimeSetting ShowGitWordColoring { get; } = new(RootSettingsPath, nameof(ShowGitWordColoring), false);
```

Comment:
[doc] missing xml-doc

Reply:
```
       /// <summary>
       /// Enable git-word-diff instead of normal "patch" viewer.
       /// </summary>
       public static BoolRuntimeSetting ShowGitWordColoring { get; } = new(RootSettingsPath, nameof(ShowGitWordColoring), false);
```